src/models/AdminTask/WhereClauseBuilder/OfferExpirationDate.ts

```diff
@@ -0,0 +1,41 @@
+import { Secretary } from "$models/Admin";
+import { ApprovalStatus } from "$models/ApprovalStatus";
+import { AdminTaskType } from "$models/AdminTask/Model";
+import { Offer } from "$models";
+
+export const OfferExpirationDateWhereClause = {
+  build: ({ secretary, modelName }: IStatusWhereClauseProps) => {
```

Comment:
esto tal vez amerite una modificacion en un indice, correte un explain de la query de admin tasks con `SET ENABLE_SEQSCAN TO OFF;` en la primer linea

Reply:
a ver si usa indices y cómo

Reply:
Se resolvio a discutir en detalle con beiro:
https://fiuba-laboral-v2.atlassian.net/browse/TPP-524